src/app/views/grants/list.js

```diff
@@ -1,89 +1,35 @@
+'use strict';
+
 var $ = require('jquery'),
     _ = require('lodash'),
-    Chartist = require('chartist'),
     Backbone = require('backbone'),
     numeral = require('numeral'),
-    Grants = require('../../models/grants'),
     template = require('../../templates/grants/list.html'),
     util = require('../../util');
 
 var GrantListView = Backbone.View.extend({
 
   template: template,
 
-  /**console.log('Loading a web page');
-var page = require('webpage').create();
-var url = 'http://phantomjs.org/';
-page.open(url, function (status) {
-  //Page is loaded!
-  phantom.exit();
-});
-
-   * Initialize the grant list
-   * @param  {Object} options
-   *                  options.direction: required. Specifies which grants to
-   *                  list. Valid values: funded, recieved
-   */
   initialize: function(options) {
-    _.bindAll(this, 'prep', 'group', 'render', 'afterRender');
-
-    var _this = this;
-
-    this.render = _.wrap(this.render, function(render) {
-      render();
-      _this.afterRender();
-      return _this;
-    });
+    _.bindAll(this, 'prep', 'group', 'render');
 
     this.direction = options.direction;
+    this.$el = options.$parent;
 
-    // Get the organziations
-    this.grants = new Grants.Collection({
-      org: options.org,
-      direction: options.direction,
-      limit: options.limit
-    });
-    this.grants.on('reset', this.render);
+    this.collection.on('reset', this.render);
```

Comment:
i'm getting a bug here due to `this.collection` being undefined -- try going to any grant page, like http://localhost:8080/#!/grants/10513